### DISCUSSION TITLE
Allow negative xp in the Event "Give Experience" option similar to the "Restore HP" or "Restore MP"

### DISCUSSION THREAD

**Discussion by PyroTech03** [AscensionGameDev/Intersect-Engine#1816] [Ideas]
This would allow us to use experience as currency or as a stricter penalty in some cases.  This is achievable by setting a negative variable, but could be made easier by allowing it within the system.

**Comment by WeylonSantana** [comment]
and this is a very easy thing to do

**Comment by WeylonSantana** [comment]
done by me in https://github.com/AscensionGameDev/Intersect-Engine/pull/2492
